lib/src/rules/avoid_redundant_argument_values.dart

```diff
@@ -9,7 +9,9 @@ import '../analyzer.dart';
 
 const _desc = r'Avoid redundant argument values.';
 
-const _details = r'''Avoid redundant argument values.
+const _details = r'''
+
+Avoid redundant argument values.
```

Comment:
This looks like an accidental copying of the `_desc` into the `_details` (at some previous time). Perhaps we should remove it.

Reply:
Done